controllers/pullRequestsController.js

```diff
@@ -0,0 +1,46 @@
+const logger = require('../utils/logger')
+const config = require('config')
+const githubService = require('../services/githubService')
+const { fetchUser } = require('../models/users')
+
+/**
+ * Fetches the pull requests in Real-Dev-Squad by user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getPullRequests = async (req, res) => {
+  try {
+    const { user } = await fetchUser(req.params.id)
+    const url = `${config.githubApi.baseUrl}/search/issues?q=org:${config.githubApi.org}+author:${user.github_id}+type:pr`
+    const { data } = await githubService.fetch(url)
```

Comment:
Why is the fetch function moved to githubService?
As per we discussed, we were to move it to /utils directory.

Also, the API call should be made from githubService file as per mentioned in the comment: https://github.com/Real-Dev-Squad/website-backend/pull/90#discussion_r531956519

The controller should handle any response modifications and error handling.